html/includes/common/worldmap.inc.php

```diff
@@ -121,6 +121,8 @@
         }
         if (!empty($widget_settings['group_radius'])) {
             $group_radius = $widget_settings['group_radius'];
+        } elseif (!empty($config['custom_worldmap'])) {
+            $group_radius = $config['custom_worldmap']['group_radius'];
         } else {
             $group_radius = 80;
```

Comment:
You can use this instead  (Config is our new config access class):
```
use LibreNMS\Config;
if (!empty($widget_settings['group_radius'])) {
    $group_radius = $widget_settings['group_radius'];
} else {
    $group_radius = Config::get('leaflet.group_radius', 80);
}
```

You also need to add it to the docs.  https://docs.librenms.org/Extensions/World-Map/

Looks like some other leaflet settings are missing there if you don't mind adding them.